src/core/qgstaskmanager.cpp

```diff
@@ -254,7 +264,7 @@ void QgsTask::setProgress( double progress )
 void QgsTask::completed()
 {
   mStatus = Complete;
-  processSubTasksForCompletion();
+  QMetaObject::invokeMethod( this, &QgsTask::processSubTasksForCompletion, Qt::AutoConnection );
```

Comment:
If I'm not wrong, AutoConnection is the default anyway.

Reply:
Right! and the call with a method pointer doesn't exist yet in Qt 5.9 (failing the CI). I changed it.